projects/ui-components/lf-selection-list/lf-selection-list.component.spec.ts

```diff
@@ -187,13 +187,13 @@ describe('LfListComponent single select', () => {
       
       // Act
       component.list?.focus();
-      tick();
+      flush();
 
       // Assert
       expect(document.activeElement).toEqual(focusItem);
     }));
 
-    it('should be able to move the focus with the arrow keys', fakeAsync(() => {
+    it('should be able to move the focus with the arrow keys', fakeAsync(async () => {
```

Comment:
why is it async?

Reply:
ahh that was before I found flush, removed